tests/test_download_cli_integration.py

```
@@ -0,0 +1,760 @@
+    integration.config = {"DOWNLOAD_DIR": "/tmp"}
+
+    result = integration.get_environment_info()
+
+    assert "python_version" in result
+    assert "working_directory" in result
+    assert "download_directory" in result
+    assert "configuration_loaded" in result
+    assert result["configuration_loaded"] is True
+
+
+def test_get_existing_prerelease_dirs(mocker):
```

Ruff (0.14.8)

Unused function argument: `mocker`

(ARG001)